bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -36,7 +36,7 @@ trait Asterism {
     * only possibilities.
     * TODO: should we Church encode this instead?
     */
-  def targets: Target \/ (SiderealTarget, SiderealTarget)
+  def targets: Target \/ (Target, Target)
```

Comment:
I understood we would be removing this altogether?  At any rate, since the asterisms will need to hold mutable `SPTarget`s, I'm not sure how to guarantee that they are and always will be sidereal.